### PR TITLE
testnet4

### PR DESCRIPTION
**Pull request by npq7721**
add setgenerate command
add future fee into future tx payload to validate when block is rescan
add future fee UI into send
add missing code from the dash 17 merged

### CODE REVIEW

File: src/evo/providertx.h [hunk 1]
```diff
@@ -268,7 +269,8 @@ class CFutureTx {
         READWRITE(maturity);
 		READWRITE(lockTime);
 		READWRITE(lockOutputIndex);
-		READWRITE(updatableByDestination);
+        READWRITE(fee);
```
Comment (CodeIsTheKey) [inline src/evo/providertx.h:261]: Check tabs in this file - looks like a mix.

File: src/Makefile.qt.include [hunk 2]
```diff
@@ -193,6 +193,8 @@ BITCOIN_QT_H = \
   qt/winshutdownmonitor.h
 
 RES_ICONS = \
+  qt/res/icons/about.png \
```
Comment (CodeIsTheKey) [inline src/Makefile.qt.include:196]: Are these two images used?  I never found a reference.

File: src/qt/sendcoinsdialog.cpp [hunk 3]
```diff
@@ -484,7 +516,9 @@ void SendCoinsDialog::accept()
 
 SendCoinsEntry *SendCoinsDialog::addEntry()
 {
-    SendCoinsEntry* entry = new SendCoinsEntry(this);
+
+    SendCoinsEntry* entry = new SendCoinsEntry(this, sporkManager.IsSporkActive(SPORK_22_SPEICAL_TX_FEE)
```
Comment (CodeIsTheKey) [inline src/qt/sendcoinsdialog.cpp]: Typo - "SPEICAL"

**Review by CodeIsTheKey — APPROVED** [review]
LGTM - just a couple notes.

File: src/Makefile.qt.include (hunk 2 above)
Comment (npq7721) [reply src/Makefile.qt.include:196]: it use in bitcoinui.cpp. i added it back

File: src/qt/sendcoinsdialog.cpp (hunk 3 above)
Comment (npq7721) [reply src/qt/sendcoinsdialog.cpp]: fixed

**Comment by CodeIsTheKey** [comment]
Looks good!

**Pull request merged**